htdocs/src/Oc/FieldNotes/Import/FileParser.php

```diff
@@ -42,34 +42,58 @@ public function parseFile(File $file): array
 
     private function getRowsFromCsv(Reader $csv): array
     {
-        $lines = [];
-        $rows = [];
         $content = $csv->getContent();
-        $content = mb_convert_encoding($content, 'UTF-8', 'UTF-16LE');
+        $content = $this->decodeToUtf8($content);
+        $rows = $this->parseCSV($content);
+        return $rows;
+    }
+
+    private function decodeToUtf8(string $input): string
+    {
+        if (!isset($input[2])) {
+            throw new FileFormatException('Input to short');
```

Comment:
spelling ... "too short"

Reply:
Are these the only logtypes we care:

```
private const FILE_LOG_TYPE_MAPPING = [
        'found it' => self::FOUND,
        "didn't find it" => self::NOT_FOUND,
        'write note' => self::NOTE,
        'needs maintenance' => self::NEEDS_MAINTENANCE,
        'owner maintenance' => self::NOTE,
```

How about "Attended"? There may be more, I haven't checked though